### PR TITLE
removing credential format specific parameters from Offer Credential in Appendix

### LINKED ISSUE #101: Remnant of PR#86
https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#name-credential-offer-3 includes `credential_definition` as required which is probably a remnant of PR #86

### PR DESCRIPTION
**Pull request by paulbastian**
Closes #101 

Work left over from #86 
We could even delete the whole Credential Offer chapters from the Appendix.

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -1720,11 +1720,6 @@ The following is a non-normative example of an object comprising `credentials_su
 
 #### Credential Offer
 
-The following additional claims are defined for this Credential format to be added to the object entry of the `credentials` parameter defined in (#credential_offer_parameters).
-
-* `credential_definition`: REQUIRED. Object containing the detailed description of the credential type. It consists at least of the following sub claims:
-  * `type`: REQUIRED. Array as defined in (#server_metadata_jwt_vc_json). This claim contains the type values the Wallet shall request in the subsequent Credential Request.
-
 The following is a non-normative example of an object comprising `credentials_supported` parameter of Credential format `jwt_vc_json`:
```
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:1723]: It's not a new problem but I think this sentence is incorrect?

```suggestion
The following is a non-normative example of a Credential Offer of type `jwt_vc_json`:
```

**Review by jogu — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1723]: `jwt_vc_json` is not credential type, it is credential format.

**Review by Sakurann — APPROVED** [review]
I think keeping the examples is good as opposed to removing credential offer sections from credential format appendix entirely.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (jogu) [reply openid-4-verifiable-credential-issuance-1_0.md:1723]: Okay, that's a good point and I think that means some of the others are wrong then? But for starters would this be right:

```suggestion
The following is a non-normative example of a Credential Offer for Credential format `jwt_vc_json`:
```

That example is definitely not an example of `credentials_supported` so the current sentence is definitely wrong.

Comment (paulbastian) [reply openid-4-verifiable-credential-issuance-1_0.md:1723]: I think you are correct, good catch

Comment (paulbastian) [reply openid-4-verifiable-credential-issuance-1_0.md]: resolved

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -1720,12 +1720,7 @@ The following is a non-normative example of an object comprising `credentials_su
 
 #### Credential Offer
 
-The following additional claims are defined for this Credential format to be added to the object entry of the `credentials` parameter defined in (#credential_offer_parameters).
-
-* `credential_definition`: REQUIRED. Object containing the detailed description of the credential type. It consists at least of the following sub claims:
-  * `type`: REQUIRED. Array as defined in (#server_metadata_jwt_vc_json). This claim contains the type values the Wallet shall request in the subsequent Credential Request.
-
-The following is a non-normative example of an object comprising `credentials_supported` parameter of Credential format `jwt_vc_json`:
+The following is a non-normative example of a Credential Offer of format `jwt_vc_json`:
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1723]:
```suggestion
The following is a non-normative example of a Credential Offer of Credential format `jwt_vc_json`:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -1796,13 +1791,7 @@ The following is a non-normative example of an object comprising `credentials_su
...
 #### Credential Offer {#issuer_initiated_issuance_ldp_vc}
 
-The following additional claims are defined for this Credential format to be added to the object entry of the `credentials` parameter defined in (#credential_offer_parameters).
-
-* `credential_definition`: REQUIRED. Object containing (and isolating) the detailed description of the credential type. It consists of the following sub claims:
-    * `@context`: REQUIRED. Array as defined in (#server_metadata_ldp_vc).
-    * `type`: REQUIRED. Array as defined in (#server_metadata_ldp_vc). This claim contains the type values the Wallet shall request in the subsequent Credential Request.
-
-The following is a non-normative example of a Credential Offer of type `ldp_vc`:
+The following is a non-normative example of a Credential Offer of format `ldp_vc`:
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1794]:
```suggestion
The following is a non-normative example of a Credential Offer of Credential format `ldp_vc`:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 4]
```diff
@@ -1897,11 +1886,7 @@ The following is a non-normative example of an object comprising `credentials_su
 
 ### Credential Offer
 
-The following additional claims are defined for this Credential format to be added to the object entry of the `credentials` parameter defined in (#credential_offer_parameters). 
-
-* `doctype`: REQUIRED. String as defined in (#server_metadata_mso_mdoc). 
-
-The following is a non-normative example of a Credential Offer of type `mso_mdoc`:
+The following is a non-normative example of a Credential Offer of format `mso_mdoc`:
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1889]:
```suggestion
The following is a non-normative example of a Credential Offer of Credential format `mso_mdoc`:
```

**Comment by Sakurann** [comment]
i don't know if `format` -> `credential format` is redundant but it is more accurate...

**Pull request merged**